doc/classes/MultiMesh.xml

```diff
@@ -94,6 +94,7 @@
 			<return type="void" />
 			<param index="0" name="instance" type="int" />
 			<param index="1" name="transform" type="Transform3D" />
+			<param index="2" name="teleport" type="bool" default="false" />
```

Comment:
Adding new arguments to a public method require binding compatibility methods for GDExtension, see https://docs.godotengine.org/en/stable/contributing/development/handling_compatibility_breakages.html

Reply:
Already fixed.
Sorry about missing that, it would appear my git pre commit hooks are outdated. Been a while since I had enough time to upstream something.